$NetBSD: patch-configure.ac,v 1.1 2012/07/28 17:47:36 gregoire Exp $

Strengthen the test for working -nostdinc -isystem.  Otherwise,
configure believes that -nostdinc -isystem works with gcc44, which
makes the build fail afterwards (for a file including stddef.h).

--- configure.ac.orig	2012-07-28 17:41:28.000000000 +0000
+++ configure.ac
@@ -717,6 +717,7 @@ AC_CACHE_CHECK([whether -nostdinc -isyst
   SAVED_CPPFLAGS="$CPPFLAGS"
   CPPFLAGS="$TARGET_CPPFLAGS -nostdinc -isystem `$TARGET_CC -print-file-name=include`"
   AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <stdarg.h>
+#include <stddef.h>
 int va_arg_func (int fixed, va_list args);]], [[]])],
       [grub_cv_cc_isystem=yes],
       [grub_cv_cc_isystem=no])
